mock/mockimpl/defaultcluster.go

```diff
@@ -6,15 +6,22 @@ import (
 )
 
 // NewDefaultCluster returns a new cluster configured with some defaults.
-func NewDefaultCluster() (mock.Cluster, error) {
+func NewDefaultCluster(port ...int) (mock.Cluster, error) {
```

Comment:
Why is this variadic? We seem to only use the first port and then just increment from that so the other ports defined are not used.

Reply:
I think that's probably better yeah, it's a bit more ugly but less misleading.